ddtrace/ddtrace.go

```diff
@@ -79,6 +79,17 @@ type SpanContext interface {
 	ForeachBaggageItem(handler func(k, v string) bool)
 }
 
+// ReadWriteSpan implementations are spans with read/write access.
+type ReadWriteSpan interface {
```

Comment:
Java calls this a `MutableSpan` instead of a `ReadWriteSpan` which I like better :)

Reply:
This sounds good to me, although it doesn't make it explicit that the span is Readable. 
cc @gbbr to have a third opinion, thoughts on naming this `MutableSpan` ?

Reply:
I don't have any strong preference. In OpenTelemetry it's called `ReadWriteSpan`, that's why I suggested this name. I think both are OK but IMHO "mutable" is more of a "Java" term and in Go it's more common to use read/write (like the many interfaces bearing this name). I have not seen this word used anywhere in the Go language.

Reply:
You're right @gbbr 👍 
e.g. https://pkg.go.dev/io#ReadWriteCloser